modules/aerospike/aerospike_test.go

```diff
@@ -0,0 +1,73 @@
+package aerospike_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	tcaerospike "github.com/testcontainers/testcontainers-go/modules/aerospike"
+)
+
+const (
+	aerospikeImage = "aerospike/aerospike-server:latest"
+)
+
+// TestAerospike tests the Aerospike container functionality
+// It includes tests for starting the container with a valid image,
+// applying container customizations, and handling context cancellation.
+// It also includes a test for an invalid image to ensure proper error handling.
+// The tests use the testcontainers-go library to manage container lifecycle
+func TestAeroSpike(t *testing.T) {
+	t.Run("invalid-image-fails", func(t *testing.T) {
+		ctx := context.Background()
+		_, err := tcaerospike.Run(ctx, "invalid-aerospike-image")
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "failed to start Aerospike container")
+	})
+
+	t.Run("valid-image-succeeds", func(t *testing.T) {
+		ctx := context.Background()
+		container, err := tcaerospike.Run(ctx, aerospikeImage)
+		require.NoError(t, err)
+		require.NotNil(t, container)
+
+		defer func() {
+			err := container.Terminate(ctx)
+			require.NoError(t, err)
+		}()
```

Comment:
suggestion: let's newest APIs for container termination in tests
```suggestion
		testcontainers.CleanupContainer(t, container)
		require.NoError(t, err)
		require.NotNil(t, container)
```